docs/migrating.md

```diff
@@ -1 +1,84 @@
+| `SIGNALFX_TRACE_{0}_ENABLED`         | Configuration pattern for enabling or disabling a specific library instrumentation. For example, in order to disable Kafka instrumentation, set `SIGNALFX_TRACE_Kafka_ENABLED=false`                                                       | `true`    | `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS` (e.g `OTEL_DOTNET_AUTO_TRACES_DISABLED_INSTRUMENTATIONS`=MongoDB) |
+
+## Exporter settings
+
+Use following settings to configure where and how the telemetry data is being exported.
+
+| Setting                                  | Description                                                                                                                                                                                           | Default                              | OpenTelemetry equivalent                                            |
+|------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|---------------------------------------------------------------------|
+| `SIGNALFX_ACCESS_TOKEN`                  | Splunk Observability Cloud access token for your organization. It enables sending telemetry directly to the Splunk Observability Cloud ingest endpoint.                                               |                                      | `SPLUNK_ACCESS_TOKEN`                                               |
+| `SIGNALFX_REALM`                         | Your Splunk Observability Cloud realm. To find your realm, open Splunk Observability Cloud, click Settings, then click on your username.                                                              | `none` (local collector)             | `SPLUNK_REALM`                                                      |
+| `SIGNALFX_ENDPOINT_URL`                  | The URL to where trace exporters send traces. Overrides `SIGNALFX_REALM` configuration for the traces ingestion endpoint.                                                                             | `http://localhost:9411/api/v2/spans` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`  |
+| `SIGNALFX_METRICS_ENDPOINT_URL`          | The URL to where metric exporters send metrics. Overrides `SIGNALFX_REALM` configuration for the metrics ingestion endpoint.                                                                          | `http://localhost:9943/v2/datapoint` | `OTEL_EXPORTER_OTLP_ENDPOINT`/`OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` |
```

Comment:
As I remember `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT` nor `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT` are not yet supported by OTel .NET SDK